src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -488,7 +514,9 @@ def make(self, key):
             parent_sorting, parent_merge_groups, quality_metrics, merge_params)
         if units_merged:
             # get merged sorting extractor here
-            return NotImplementedError
+            sorting = MergedSortingExtractor(parent_sorting=parent_sorting, merge_groups=merge_groups)
```

Comment:
`sorting` is never assigned anything. Was it intended that this be assigned to `parent_sorting`?